test/middleware/spec.back-link.js

```diff
@@ -168,6 +175,7 @@ describe('Back Links', function () {
 
     it('returns undefined if referrer header is not on whitelist', function () {
         req.get.withArgs('referrer').returns('http://example.com/not-whitelisted');
+        req.baseUrl = '/not-whitelisted';
```

Comment:
It is stubbing the shortcomings of `req.get.withArgs('referrer').returns('http://example.com/not-whitelisted');` The shortcoming being that the stub does not assign a value to `req.baseUrl`.
IMO adding a baseUrl value avoids assuming too much.